src/Sql/Sql/ManagedDatabase/Cmdlet/StartAzureSqlInstanceDatabaseLogReplay.cs

```diff
@@ -0,0 +1,135 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using Microsoft.Azure.Commands.Sql.Properties;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to start a Azure Sql Managed Database Log Replay
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Start, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDatabaseLogReplay",
+        DefaultParameterSetName = LogReplayByNameAndResourceGroupParameterSet,
+        SupportsShouldProcess = true),
+    OutputType(typeof(AzureSqlManagedDatabaseModel))]
+    public class StartAzureSqlInstanceDatabaseLogReplay : AzureSqlManagedDatabaseLogReplayCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the URI of the storage container to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The storage container URI.")]
+        [Alias("Storage")]
+        [ValidateNotNullOrEmpty]
+        public string StorageContainerUri { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Sas token of the storage container.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The storage container Sas token.")]
+        [Alias("SasToken")]
+        [ValidateNotNullOrEmpty]
+        public string StorageContainerSasToken { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether or not to automatically complete restore upon completion.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The indicator whether or not to auto-complete restore upon completion.")]
+        public SwitchParameter AutoCompleteRestore { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the last backup.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public string LastBackupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the instance database collation to use
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The collation of the instance database to use.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("SQL_Latin1_General_CP1_CI_AS", "Latin1_General_100_CS_AS_SC")]
+        public string Collation { get; set; }
```

Comment:
You provide argument completer here but the related item in help file is not generated. Please update cmdlet the help file.

Reply:
I am not sure what you mean by saying this. I followed other examples and they have same help files when it comes to Collation property

Reply:
> I am not sure what you mean by saying this. I followed other examples and they have same help files when it comes to Collation property

Please refer to -Edition of the cmdlet https://docs.microsoft.com/en-us/powershell/module/az.sql/new-azsqldatabase?view=azps-4.6.1
The value of in argument completer will be generated as accepted values in help file.

Reply:
> I am not sure what you mean by saying this. I followed other examples and they have same help files when it comes to Collation property

Please neglect my comments here. I may misunderstanding something. I will double confirm. Please fix the other help file issues